daml-assistant/src/DA/Daml/Assistant/Install/Path.hs

```diff
@@ -76,13 +79,49 @@ regQueryStringValue key valueName =
             failUnlessSuccess "regQueryDefaultValue" $
                 c_RegQueryValueEx p_key c_valueName nullPtr nullPtr c_value p_valueLen
             peekTString (castPtr c_value)
-#else
-updatePath _ output targetPath = suggestAddToPath output targetPath
-#endif
 
 suggestAddToPath :: (String -> IO ()) -> FilePath -> IO ()
 suggestAddToPath output targetPath = do
     -- Ask user to add .daml/bin to PATH if it is absent.
     searchPaths <- map dropTrailingPathSeparator <$> getSearchPath
     when (targetPath `notElem` searchPaths) $ do
         output ("Please add " <> targetPath <> " to your PATH.")
+#else
+updatePath _ output targetPath = do
```

Comment:
It would be good to have a command-line argument to turn off the interactive prompt (and passed to here via the first argument, `InstallOptions`), just in case.

Reply:
I'll add it.

Reply:
Actually, I reuse the already present `iSetPath` option of the InstallOptions.

Reply:
Ah yeah, I just remembered this option already exists. Good call!

I think we should change iSetPath to be an underlying `YesNoAuto` instead of `Bool`, and only run the interactive prompt when `auto` is used (the default). Presumably, if the user supplies the explicit `--install-path=yes` then they do want the path to be installed, and if they supply `--install-path=no` then they don't, no need to ask.

(Maybe in a separate PR.)

Reply:
I changed it now.

Reply:
Thank you very much, looks great!